src/uwtools/utils/cli_helpers.py

```diff
@@ -51,35 +50,7 @@ def path_if_it_exists(path: str) -> str:
     p = Path(path)
     if not p.exists():
         msg = f"{path} does not exist"
-        logging.critical(msg)
```

Comment:
`logging` cannot be used here because this function is called from `parse_args()` functions, so logging has not been set up yet. Use `print(..., file=sys.stderr)` instead.